pkg/build/fuchsia.go

```diff
@@ -14,15 +16,35 @@ import (
 
 type fuchsia struct{}
 
+// syzRoot returns $GOPATH/src/github.com/google/syzkaller.
+func syzRoot() (string, error) {
+	_, selfPath, _, ok := runtime.Caller(0)
```

Comment:
This is quite an implicit dependency. syz-ci/updater.go may run git pull there at this very moment.
But I guess we can live with this for now and may be this will break less often than out-of-tree fuchsia builds :)